app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -25,6 +25,7 @@ import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
 import androidx.test.espresso.idling.CountingIdlingResource
 import androidx.test.espresso.intent.Intents
 import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.hasChildCount
```

Comment:
wildcard import at line number 3. please have a look.

Reply:
Not wildcard I guess, it should be reformated as java, javax, and kotlin import comes at the end of the line

Reply:
I can't seem to find any wildcard import at line no 3

Reply:
okay doing that